### ISSUE TITLE
windows triton 加载VoxCPM模型失败: Python int too large to convert to C long

### ISSUE THREAD

**Issue by maxin9966** [OpenBMB/VoxCPM#27]
PyTorch 版本: 2.8.0+cu129
CUDA 是否可用: True
检测到的 GPU 数量: 1
当前 GPU 型号: NVIDIA GeForce RTX 4060 Laptop GPU
Fetching 12 files: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████| 12/12 [00:00<?, ?it/s]
voxcpm_model_path: C:\Users\taiao\.cache\huggingface\hub\models--openbmb--VoxCPM-0.5B\snapshots\f67d35a3848e0bec0fdb8c33e6fc92cf293ee72f, zipenhancer_model_path: iic/speech_zipenhancer_ans_multiloss_16k_base, enable_denoiser: True
D:\ai\ai-test\gui\.venv\Lib\site-packages\torch\nn\utils\weight_norm.py:144: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
D:\ai\ai-test\gui\.venv\Lib\site-packages\modelscope\utils\plugins.py:18: UserWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
  import pkg_resources
INFO:datasets:PyTorch version 2.8.0+cu129 available.
Downloading Model from https://www.modelscope.cn to directory: C:\Users\taiao\.cache\modelscope\hub\models\iic\speech_zipenhancer_ans_multiloss_16k_base
2025-09-20 12:56:11,919 - modelscope - INFO - initiate model from C:\Users\taiao\.cache\modelscope\hub\models\iic\speech_zipenhancer_ans_multiloss_16k_base
2025-09-20 12:56:11,919 - modelscope - INFO - initiate model from location C:\Users\taiao\.cache\modelscope\hub\models\iic\speech_zipenhancer_ans_multiloss_16k_base.
2025-09-20 12:56:11,923 - modelscope - INFO - initialize model from C:\Users\taiao\.cache\modelscope\hub\models\iic\speech_zipenhancer_ans_multiloss_16k_base
2025-09-20 12:56:12,077 - modelscope - WARNING - No preprocessor field found in cfg.
2025-09-20 12:56:12,077 - modelscope - WARNING - No val key and type key found in preprocessor domain of configuration.json file.
2025-09-20 12:56:12,077 - modelscope - WARNING - Cannot find available config to build preprocessor at mode inference, current config: {'model_dir': 'C:\\Users\\taiao\\.cache\\modelscope\\hub\\models\\iic\\speech_zipenhancer_ans_multiloss_16k_base'}. trying to build by task and model information.
2025-09-20 12:56:12,077 - modelscope - INFO - No preprocessor key ('speech_zipenhancer_ans_multiloss_16k_base', 'acoustic-noise-suppression') found in PREPROCESSOR_MAP, skip building preprocessor. If the pipeline runs normally, please ignore this log.
Warm up VoxCPMModel...
  0%|                                                                                                                               | 0/10 [00:02<?, ?it/s]
加载VoxCPM模型失败: Python int too large to convert to C long

**Comment by wangfeng35** [comment]
这个是pytorch的问题，需要修改`torch_python.dll`文件。
详细见：https://github.com/pytorch/pytorch/issues/162430#issuecomment-3289054096
